index.js

```diff
@@ -148,14 +148,19 @@ function getCombinedBranch(branchConfig) {
   return branchName;
 }
 
-async function handleSwitchToBranchCommand(sg, sortedBranches, combinedBranch) {
+async function handleSwitchToBranchCommand(sg, currentBranch, sortedBranches, combinedBranch) {
   const switchToBranchIndex = program.args[0];
   if (typeof switchToBranchIndex === 'undefined') {
     return;
   }
   let targetBranch;
+  let currentBranchIndex = sortedBranches.indexOf(currentBranch)
```

Comment:
When contributing to another repository, it is important to maintain the existing style. You are free to adopt whatever style you like in your own repos.

In this case, the repo uses semicolons, so all your changes should do the same.
```suggestion
  let currentBranchIndex = sortedBranches.indexOf(currentBranch);
```